packages/now-go/index.ts

```diff
@@ -1,50 +1,59 @@
-const { join, sep, dirname } = require('path');
-const {
+import { join, sep, dirname } from 'path';
+import {
   readFile, writeFile, pathExists, move,
-} = require('fs-extra');
+} from 'fs-extra';
 
-const glob = require('@now/build-utils/fs/glob.js'); // eslint-disable-line import/no-extraneous-dependencies
-const download = require('@now/build-utils/fs/download.js'); // eslint-disable-line import/no-extraneous-dependencies
-const { createLambda } = require('@now/build-utils/lambda.js'); // eslint-disable-line import/no-extraneous-dependencies
-const getWritableDirectory = require('@now/build-utils/fs/get-writable-directory.js'); // eslint-disable-line import/no-extraneous-dependencies
-const { createGo, getExportedFunctionName } = require('./go-helpers');
+import {
+  glob, download, createLambda, getWriteableDirectory, BuildOptions, shouldServe,
+} from '@now/build-utils';
 
-const config = {
+import { createGo, getAnalysedEntrypoint } from './go-helpers';
+
+interface Analysed {
+  packageName: string;
+  functionName: string;
+  watch: string[];
+}
+
+export const version = 2;
+
+export const config = {
   maxLambdaSize: '10mb',
 };
 
-async function build({ files, entrypoint }) {
+export async function build({ files, entrypoint }: BuildOptions) {
   console.log('Downloading user files...');
 
   const [goPath, outDir] = await Promise.all([
-    getWritableDirectory(),
-    getWritableDirectory(),
+    getWriteableDirectory(),
+    getWriteableDirectory(),
   ]);
 
   const srcPath = join(goPath, 'src', 'lambda');
   const downloadedFiles = await download(files, srcPath);
 
   console.log(`Parsing AST for "${entrypoint}"`);
-  let parseFunctionName;
+  let analysed;
```

Comment:
```suggestion
  let analyzed;
```

Can we add a type here instead of implicit any?